package.json

```diff
@@ -56,6 +56,7 @@
     "test": "cross-env NODE_ICU_DATA=./vendors/node_modules/full-icu jest --config jest.test.config.js",
     "test:watch": "cross-env NODE_ICU_DATA=./vendors/node_modules/full-icu jest --config jest.test.config.js --watch",
     "test:bundle": "jest --config jest.bundle.config.js",
+    "test:visual": "jest --config jest.visual.config.js",
```

Comment:
now that we are using our visual tests to test the rich text editor's behaviour, it makes sense to have this as it's own command